README.md

```diff
@@ -4,27 +4,110 @@ This document describes how you can collect Corelight Sensor logs by configuring
 
 For more information, see [Data ingestion to Chronicle](https://cloud.google.com/chronicle/docs/data-ingestion-flow).
 
-The following deployment architecture diagram shows how a Corelight Sensor is configured to send logs to  Chronicle . Each customer deployment might differ from this representation and might be more complex.
+## Before you begin
+
+*  Verify the version of Corelight Sensor. The Corelight Google SecOps parser was designed for version 27.12 and earlier. Later versions of the Corelight Sensor might have additional logs that the parser won't recognize, and those logs might receive limited or no field parsing. However, the log content will still be available in the raw log format in Google SecOps.
+*  Ensure that all systems in the deployment architecture are configured with the UTC time zone.
+
+
+## Deployment and Log Ingestion Methods
+
+The following deployment architecture diagram illustrates how a Corelight Sensor is set up to send logs to Google Security Operations using two different ingestion architectures. It's important to note that each customer deployment may vary from this representation and could be more complex.
+
+An ingestion label identifies the parser which normalizes raw log data
+to structured UDM format. The information in this document applies to the parser
+with the `CORELIGHT` ingestion label.
 
-![Deployment architecture](images/corelight_parser_arch.png)
+## Ingesting Logs into Google SecOps using Corelight Exporters 
+
+![Deployment architecture](images/parser-corelight-architecture-without-forwarder.png)
 
 The architecture diagram shows the following components:
 
-* **Corelight Sensor**: The system running the [Corelight Sensor](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html).
+*  **Corelight Sensor**: The system running the [Corelight Sensor ](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html).
```

Comment:
Please change this line to:

*  **Corelight Sensor**: The system running the [Corelight Sensor ](https://docs.corelight.com/docs/sensor/sensor/quickstarts/quickstart.html).

Reply:
Done.